authors/apps/authentication/tests/test_user_registration.py

```diff
@@ -137,3 +166,49 @@ def test_register_duplicate_username(self):
                                                         ["user with this"
                                                          " username already"
                                                          " exists."]}})
+
+    def test_username_special_characters_only(self):
```

Comment:
test names should say what the test is doing, example: `test_registration_rejects_username_with_special_characters_only`

Reply:
Okay noted.